magento/inventory
# 1539 move source dropdown values to a separated class

### Description

1. Sorce dropdown values should be in a separated class
2. Source Selection Algorithm in \Magento\InventoryShippingAdminUi\Ui\DataProvider\SourceSelectionDataProvider should be the same as in `\Magento\InventoryShippingAdminUi\Controller\Adminhtml\SourceSelection\ProcessAlgorithm`

### Fixed Issues (if relevant)

1. https://github.com/magento-engcom/msi/issues/1539: Refactoring: Move the Source dropdown values to a separate class

### Manual testing scenarios

1. Assign the Sources to the products
2. Create a Order with 2+ items
3. In the Order view Press "Ship"
4. "Source Selection" page should open.
Source dropdown shoud have sorted sources

### Contribution checklist
